### PR TITLE
Add ability to set minimum Puma threads with MIN_THREADS environment variable

### PR DESCRIPTION
**Pull request by jimeh**

### CODE REVIEW

**Comment by matthewford** [comment]
Looks good to me

**Review by ykzts — APPROVED** [review]

**Comment by Gargron** [comment]
Moving the Helm chart out to mastodon/chart

File: config/puma.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 persistent_timeout ENV.fetch('PERSISTENT_TIMEOUT') { 20 }.to_i
 
-threads_count = ENV.fetch('MAX_THREADS') { 5 }.to_i
-threads threads_count, threads_count
+min_threads_count = ENV.fetch('MIN_THREADS') { 5 }.to_i
```
Comment (Gargron) [inline config/puma.rb:3]: This changes behaviour, perhaps fallback to `MAX_THREADS` value instead? Unless changing the behaviour is a performance improvement of some sort.

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/puma.rb (hunk 1 above)
Comment (jimeh) [reply config/puma.rb]: You're right, good catch. Fixed it to default to `max_thread_count` as you suggested. This should ensure identical behavior if `MIN_THREADS` is not set.

**Review by Gargron — APPROVED** [review]

**Pull request merged**